code/ebike.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>BikeMyLife</title>
+    <link rel="stylesheet" type="text/css" href="styleproject.css"/>
+
+    <!-- If you're using google fonts, that should be added here -->
+  </head>
+  <body>
+    <img src="EBike_Regen.jpg">
+    <div class="title-box">
+      <h1>Bike My Life</h1>
+    </div>
+
+  <div class="text-box">
+
+      <h2>Seite in Arbeit</h2>
+  
+      <!--Beschreibung Sinn der Seite-->
+      <p>
+      </p>    
+  </div>
+  
+  <menu>
+      <a href="./index.html">Home</a>
+      <a href="./bike.html">Bike</a>
+      <a href="./rennvelo.html">Rennvelo</a>
+      <a href="./ebike.html">E-Bike</a>
+      <a href="./ueber_mich.html">Über mich</a>
+    </menu>
```

Comment:
Watch out the indentation here! menu's closing tag should be in line with the opening tag. 😀